sdk/program/src/vote/state/vote_state_versions.rs

```diff
@@ -1,8 +1,9 @@
-use super::{vote_state_0_23_5::VoteState0_23_5, *};
+use super::{vote_state_0_23_5::VoteState0_23_5, vote_state_1_10_40::VoteState1_10_40, *};
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
 pub enum VoteStateVersions {
     V0_23_5(Box<VoteState0_23_5>),
```

Comment:
@carllin can we remove this super old one now? This migration's been live for a while now right?

Reply:
I think this is a good idea.  Less to maintain would definitely be a win.

Reply:
the enum can't shift

Reply:
This is true, but there is a lot of code associated with supporting it that could be removed.